projects/geo/src/lib/datasource/shared/capabilities.service.ts

```diff
@@ -4,7 +4,7 @@ import { Observable, forkJoin } from 'rxjs';
 import { map } from 'rxjs/operators';
 
 import { WMSCapabilities, WMTSCapabilities } from 'ol/format';
-import olSourceWMTS from 'ol/source/WMTS';
+import olSourceWMTS, {optionsFromCapabilities} from 'ol/source/WMTS.js';
```

Comment:
olSourceWMTS import can be removed.

```suggestion
import { optionsFromCapabilities } from 'ol/source/WMTS.js';
```